datastore/db/bytebase/FertiScan/new_inspection/new_organization_located.sql

```diff
@@ -0,0 +1,40 @@
+CREATE OR REPLACE FUNCTION "fertiscan_0.0.11".new_organization_located(
```

Comment:
No. I plan on making changes to the update of the DB object to include a original values and the actual values. We also need to use the new function when creating the objects instead of an upsert because:
1 it enables us to better manage the creation of objects
2 it enables us to include multiple layers of the DB into the new functions

Reply:
In a separate issue right?